### PR TITLE
Implement LNet Status Indicator component

### LINKED ISSUE #1465: Implement LNet Status indicator
Implement LNet status indicator component.

### PR DESCRIPTION
**Pull request by jgrund**
Fixes #1465.

Screen Shot 2020-01-13 at 9 22 05 AM

Signed-off-by: Joe Grund <jgrund@whamcloud.io>

---
This change is [Reviewable](https://reviewable.io/reviews/whamcloud/integrated-manager-for-lustre/1466)

### CODE REVIEW

File: iml-gui/crate/src/components/lnet_status.rs [hunk 1]
```diff
@@ -0,0 +1,26 @@
+use crate::{components::font_awesome, generated::css_classes::C, MergeAttrs};
+use iml_wire_types::db::LnetConfigurationRecord;
+use seed::{prelude::*, *};
+
+fn network<T>(color: impl Into<Option<&'static str>>) -> Node<T> {
+    if let Some(color) = color.into() {
+        font_awesome(
+            class![C.w_4, C.h_4, C.inline, C.mr_1].merge_attrs(class![color]),
```
Comment (ip1981) [inline iml-gui/crate/src/components/lnet_status.rs]: I have seen in some PR, that you created a trait for merging attrs, no?

Comment (jgrund) [reply iml-gui/crate/src/components/lnet_status.rs]: pushed.

File: iml-gui/crate/src/components/lnet_status.rs [hunk 2]
```diff
@@ -0,0 +1,23 @@
+use crate::{components::font_awesome, generated::css_classes::C};
```
Comment (johnsonw) [inline iml-gui/crate/src/components/lnet_status.rs]: Copyright?

Comment (jgrund) [reply iml-gui/crate/src/components/lnet_status.rs]: Will add to all files before publishing first version.

File: iml-gui/crate/src/lib.rs [hunk 3]
```diff
@@ -57,6 +57,31 @@ impl Visibility {
     }
 }
 
+/// Allows for merging attributes onto an existing item
```
Comment (ip1981) [inline iml-gui/crate/src/lib.rs]: Not used any more?

Comment (jgrund) [reply iml-gui/crate/src/lib.rs]: Not in this PR, but I'm using it elsewhere so I figured I'd keep it here.

**Pull request merged**